modules/salt/pages/custom-states.adoc

```diff
@@ -0,0 +1,163 @@
+
+You can use the {productname} {webui} to create and edit custom Salt state files.
+You must create a state channel first, with an initial state named ``init.sls``.
+The ``init.sls`` file is used to reference all other state files within the channel.
+The custom states that you create using the {webui} are stored on the {productname} Server in the the [path]``/srv/susemanager/salt/<organization>/`` directory.
+
+After the channel is created with an ``init.sls`` file, you can write additional state files in the {webui}.
+Alternatively, you can upload existing state files to use within your state channel, or import them from other channels or clients.
+
+
+
+.Procedure: Creating a Custom Salt Channel and Initial State
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click btn:[Create State Channel].
+. In the [guimenu]``Name`` field, type a name for your state.
+. In the [guimenu]``Label`` field, type a label.
+  Use alphanumeric characters, hyphens, and underscores.
+  Do not use spaces.
+. In the [guimenu]``Description`` field, type a short description of the configuration your state performs.
+. In the [guimenu]``SLS Contents`` field, type the contents of your ``init.sls`` state.
+  Ensure your file starts by specifying the source of the managed file, using this syntax:
++
+----
+file.managed:
+  - source: salt://<org_name>/<channel_name>/etc/<ID>/<filename>
+----
++
+ Example custom state files are given later in this section.
+. Click btn:[Update Channel] to save your state.
+
+
+
+.Procedure: Adding Additional Files to a Custom State Channel
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click the name of the channel you want to add files to.
+. To create a new file, click btn:``Create configuration file`` and type the contents of the file.
+. To upload an existing file, click btn:``Upload Configuration Files`` and select the file to upload.
+. To copy an existing file, click btn:``Import a File from Another Channel or System`` and select the file to copy.
+
+
+
+.Procedure: Editing a Custom Salt State
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click btn:[View/Edit <filename>.sls File].
+. Make your changes to the file
+. Click btn:[Update Configuration File] to save your state.
+
+You can also manage revisions, compare the state to others in your organization, and download the ``.sls`` file from this dialog.
+
+
+
+.Procedure: Assigning a Client to a Custom Salt State
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click the name of the state you want to assign a client to.
+. Navigate to the menu:Systems[Target Systems] tab.
+. Check the clients you want to assign.
+. Click btn:[Subscribe systems].
+
+
+
+== Example Custom State Files
+
+This section contains some example custom state files.
+Use these as a basis for writing your own custom states.
+
+
+
+.Example: Initial State File
```

Comment:
```suggestion
.Example: Manage a File
```

Initial State File is just a state with the name of the channel. It has no special content.
It can contain whatever the customer want to manage. There is no need to manage additional files at all.

I would suggest to list here a number of the mostly used state modules.

- file.managed
- pkg.installed
- cmd.run
- service.running